astro/astro.go

```diff
@@ -45,45 +45,42 @@ type Project struct {
 }
 
 // NewProject returns a new instance of Project.
-func NewProject(config conf.Project) (*Project, error) {
+func NewProject(opts ...Option) (*Project, error) {
+	project := &Project{}
```

Comment:
Dumb question: I am curious why we switch the order of struct initialization and logging message 😜

Reply:
well, previously we were initializing before we said "initializing" so now it's more correct 😄